doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
+| ``MULTIPR_DEPENDABOT``              | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| ``PYANSYS_PYPI_PRIVATE_PAT``        | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+
+
+.. _organization_tokens:
+
+Organization tokens
+-------------------
+
+To facilitate certain tasks such as authentication, ``Ansys`` and ``Ansys-internal`` organizations provide to the developer with certain tokens or personal access tokens (PATs).
+These tokens are confidential and for internal use only.
+Some of these tokens can be used as GitHub secrets, others must be requested to pyansys.core@ansys.com.
+
+
+.. _github_tokens:
+
+GitHub tokens
+~~~~~~~~~~~~~
+These tokens are meant to be used within GitHub to provide access and permissions to different tasks and repositories.
+
+Fine-grained tokens
+*******************
+
++----------------------------------------------------------------+----------------------------------------+
+| | **TOKEN NAME**                                               | ``dependabot-multi-pr``                |
++================================================================+========================================+
+| | **Repository access**                                        |  Public Repositories (read-only)       |
++----------------------------------------------------------------+----------------------------------------+
+| | **Permissions**                                              |                                        |
++----------------------------------------------------------------+----------------------------------------+
+| |    **Block another user**                                    |  No access                             |
+| |     View and manage users blocked by the user.               |                                        |
++----------------------------------------------------------------+----------------------------------------+
+| |    **Codespaces user secrets**                               |  No access                             |
+| |     Manage Codespaces user secrets.                          |                                        |
++----------------------------------------------------------------+----------------------------------------+
+| |    **Email addresses**                                       |  No access                             |
+| |     Manage a user's email addresses.                         |                                        |
```

Comment:
```suggestion
| |     Manage user email addresses.                         |                                        |
```
Use as an adjective--otherwise you'd have to use users' email addresess (plural possessive).